### ISSUE TITLE
CONAN_CMAKE_CXX_STANDARD not set when building a workspace

### ISSUE THREAD

**Issue by sogartar** [conan-io/conan#9041]
I have a workspace with a few connan+CMake projects. In my `conanfile.py` I can see that during configuration `settings.compiler.cppstd` has the correct value from my profile. This value does not propagate to `CONAN_CMAKE_CXX_STANDARD` in CMake. Consequently `CMAKE_CXX_STANDARD` is not set.

### Environment Details
  * Operating System: Windows 10
  * Compiler: MSVC 19.29.30037
  * Conan version: 1.36
  * Python version: 3.8.6

### Steps to reproduce
I have something along the lines of
```
conan workspace install "$SRC_DIR/conan-workspace.yml --profile "$SRC_DIR/conan-msvc-profile"
cmake "$SRC_DIR"
```

**Comment by danimtb** [comment]
Hi @sogartar,

Thanks a lot for your request. Unfortunately, workspaces feature is on hold. We are currently improving the integration with generators and toolchain and this issue is related to that matter as we are focused on improving the developer experience.

**Comment by memsharded** [comment]
Hi @sogartar 

We have resumed work on Conan workspaces for Conan 2. Initial release for testing will be released in Conan 2.10 today.

This ticket is now outdated, because it was connected to the legacy build-system integrations that have been removed in Conan 2. The new ``CMake`` integrations no longer use ``CONAN_`` env-vars, but use toolchains, and ``conf`` definitions from profiles or global conf.

I am closing this ticket as outdated, we will publish the docs for the new workspaces under the "Incubating" section on the docs, looking forward for feedback, please create new tickets for any question or issue about the new workspaces. Thanks very much!
